jaraco/collections/__init__.py

```diff
@@ -186,7 +193,7 @@ class RangeMap(dict):
     which requires use of sort params and a key_match_comparator.
 
     >>> r = RangeMap({1: 'a', 4: 'b'},
-    ...     sort_params=dict(reverse=True),
+    ...     sort_params={'reverse': True},
```

Comment:
I prefer the `dict` construction. It uses a syntax that's symmetric with how the parameters are defined for the `sort` function (i.e. you don't see `sort(**{'reverse': True})`). I realize it's more characters to use the dict constructor, but it's fewer characters (cleaner) inside the dict and fewer characters overall and easier to type when this approach is used in a [larger construction](https://github.com/jaraco/jaraco.abode/blob/84b3a63a0fe3b785b54cc0ebe1abd620b6eca094/tests/mock/devices/door_lock.py#L16-L64). Most importantly, when this syntax is used outside of a docstring (or in an editor that performs syntax highlighting on Python code in docstrings), the use of the dict constructor provides better syntax highlighting.

Reply:
I just checked https://docs.astral.sh/ruff/rules/unnecessary-collection-call/ again and just saw that it is configurable specifically for that preference with https://docs.astral.sh/ruff/settings/#lintflake8-comprehensions